controllers/follow.js

```diff
@@ -70,4 +71,58 @@ export default class FollowController {
       message: 'User unfollowed successfully.'
     });
   }
+
+  /**
+   * @description Get user's followers and followings stats
+   * @param {object} req http request object
+   * @param {object} res http response object
+   * @returns {object} response
+   */
+  static async getUserFollowers(req, res) {
+    const { id } = req.user;
+    const requiredOutput = {
```

Comment:
@kleva-j since these are attributes, is `requiredOutputs` a better variable name?

Reply:
Thanks, I'll make changes to this.